pkgs/os-specific/linux/usb-blaster-udev-rules/usb-blaster.rules

```diff
@@ -0,0 +1,8 @@
+# USB-Blaster
+ATTRS{idVendor}=="09fb", ATTRS{idProduct}=="6001", MODE="666", TAG+="uaccess"
+ATTRS{idVendor}=="09fb", ATTRS{idProduct}=="6002", MODE="666", TAG+="uaccess"
```

Comment:
I don't think we want `MODE="666"`. This would give every process the ability to access the device (even for example daemons like `dhcpcd` that have no business looking at it).

I think `TAG+="uaccess"` alone should be sufficient if you want to give the logged-in user access to the device, but documentation is sparse about this. Could you try that out? I don't have access to my USB-Blaster until Monday.

Reply:
FWIW, this is the rule I use:

```
    SUBSYSTEM=="usb", ATTR{manufacturer}=="Altera", ATTR{product}=="USB-Blaster", \
      GROUP="users"
```

Reply:
yeah removing `MODE` worked great